db/migrations/0001_initial.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 4.0.2 on 2023-11-03 15:13
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Actor',
```

Comment:
change ' to ", use Black

Reply:
I don't understand at all what is meant here?

Reply:
don't use 'Actor', use "Actor". Black change it automatically

Reply:
Please, I don't understand. What is mean - " don't use 'Actor', use "Actor"."  What it is  - the "Black" ?